### ISSUE TITLE
OS Information retrieval is not possible on MacOS/Linux

### ISSUE THREAD

**Issue by adbertram** [pester/Pester#644]
This probably needs changing to use Get-OperatingSystem.

**Comment by nohwnd** [comment]
Could you be more specific about the problem please? Where does it fail? What is the error message?

**Comment by adbertram** [comment]
Yep, this is the warning that shows up whenever Pester is loaded if it can't find Get-WmiObject or Get-CimInstance.
